app/Services/PirepService.php

```diff
@@ -489,6 +476,22 @@ public function submit(Pirep $pirep)
         Log::info('New PIREP filed', [$pirep]);
         event(new PirepFiled($pirep));
 
+        // Figure out what pirep state should be, if nothing provided yet.
```

Comment:
If you add a `$pirep->refresh()` there, then someone can change it in a custom listener

Reply:
Yeah I forgot that `refresh()` :)

And yes, I think a `save()` would be needed if they decide to change things in a pirep.